docs/Middlewares.md

```diff
@@ -29,15 +29,16 @@ const helmet = require('fastify-helmet')
 fastify.register(helmet)
 ```
 
-Also remember that middlewares can be encapsulated, this means that you can decide where your middlewares should run by using `register` as explained in the [plugins guide](https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md).
+Remember that middlewares can be encapsulated, this means that you can decide where your middlewares should run by using `register` as explained in the [plugins guide](https://github.com/fastify/fastify/blob/master/docs/Plugins-Guide.md).
+
+Fastify middlewares also do not expose the `send` method or other methods specific to the Fastify [Reply]('./Reply.md' "Reply") instance. This is because Fastify wraps the incoming `req` and `res` NodeJS instances using the [Request](./Request.md "Request") and [Reply](./Reply.md "Reply") classes internally, but this is done after the middlewares phase. If you need to create a middleware you have to use the NodeJS `req` and `res` instances. Otherwise, you can use the `preHandler` hook that has the [Request](./Request.md "Request") and [Reply](./Reply.md "Reply") Fastify instances. For more information, see [Hooks](./Hooks.md "Hooks").
```

Comment:
+ "NodeJS" => "Node"
+ "Request and Reply classes" => "Request and Reply objects" (or "prototypes" instead of "objects")

Reply:
No problem